database/models/token.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Token = sequelize.define(
+    'Token',
+    {
+      id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        defaultValue: DataTypes.UUIDV4
+      },
+      userId: {
+        type: DataTypes.UUID
+      },
+      token: {
+        type: DataTypes.STRING
+      },
+      status: {
+        type: DataTypes.STRING,
+        defaultValue: 'active'
+      },
+      signoutAt: {
+        type: DataTypes.DATE
+      },
+      createdAt: {
+        allowNull: false,
+        type: DataTypes.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: DataTypes.DATE
+      }
+    },
+    {}
```

Comment:
What is the use of this empty object

Reply:
@Nziranziza It is generated by sequelize when creating a table model but can be removed. I am going to remove it